azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/features/VirtualMachineApi.java

```diff
@@ -131,5 +132,27 @@ VirtualMachine create(@PathParam("vmname") String vmname,
    @Path("/{name}/powerOff")
    void stop(@PathParam("name") String name);
 
+   /**
+    * Generalize the virtual machine
+    */
+   @Named("generalize")
+   @POST
+   @Path("/{name}/generalize")
+   @ResponseParser(StatusCodeParser.class)
```

Comment:
What is the purpose of returning the status here? What information does it provide? Can't we just return a `boolean` or `void`?